TAScheduler/UnitTests/tests.py

```diff
@@ -2946,3 +2946,94 @@ def test_none_to_remove(self):
     def test_remove(self):
         self.lecture.removeTA()
         self.assertIsNone(self.lecture.getLectureTAAsgmt(), "removeTA() did not remove from lecture")
+
+class UserEditMyContactInfoTest(TestCase):
+    def setUp(self):
+        # Create a user instance for testing
+        self.user = User.objects.create(
+            email_address='test@example.com',
+            password='safe_password',
+            first_name='Test',
+            last_name='User',
+            home_address='123 Main St',
+            phone_number=1234567890
+        )
+
+    def test_edit_contact_info(self):
+        # Simulate the edit operation
+        self.user.first_name = 'UpdatedName'
+        self.user.save()
+
+        # Fetch the updated user from the database
+        updated_user = User.objects.get(id=self.user.id)
+
+        # Check if the first name is updated
+        self.assertEqual(updated_user.first_name, 'UpdatedName')
```

Comment:
Maybe additional tests for each field as well

Reply:
Make sure when you write the editContact method you provide default values to all of the variables so that the function can actually work and not throw errors because not every things was provided. Or change the input to be a dict so that so you use what I did in editAccount under Admin to just not change values if something was not passed to the function.